install/FullInstall/fullInstall.sql

```diff
@@ -902,3 +902,23 @@ UPDATE `wD_Misc` SET `value` = '161' WHERE `name` = 'Version';
 
 ALTER TABLE `wD_Games` ADD COLUMN `playerTypes` enum('Members', 'Mixed', 'MemberVsBots') DEFAULT 'Members' NOT NULL;
 ALTER TABLE `wD_Backup_Games` ADD COLUMN `playerTypes` enum('Members', 'Mixed', 'MemberVsBots') DEFAULT 'Members' NOT NULL;
+
+ALTER TABLE `wD_Games`
+ADD COLUMN `nextPhaseMinutes` int(10) UNSIGNED DEFAULT 0 NOT NULL AFTER `phaseMinutes`,
+ADD COLUMN `phaseSwitchPeriod` int(10) DEFAULT -1 AFTER `nextPhaseMinutes`,
+ADD COLUMN `createTime` int(10) UNSIGNED DEFAULT NULL,
```

Comment:
I can't find where this new column createTime is actually used. From what I can see it isn't being utilized anywhere in the code aside from being loaded in the game object.

Reply:
Removed all references to createTime

Reply:
Thanks!